benefits/core/models.py

```diff
@@ -206,21 +206,16 @@ class PaymentProcessor(models.Model):
     id = models.AutoField(primary_key=True)
     name = models.TextField()
     api_base_url = models.TextField()
```

Comment:
I think we are OK to proceed as-is, but I wanted to raise that the `littlepay.api.Client` also has a `version` parameter. 

This defaults to `v1` so it should work. However in the future we may need to add a field to this model if we need to change the version. Maybe worth doing now? Your call.

Reply:
I hadn't thought much about that `version` parameter, so I'm glad you brought it up.

I'm thinking it would be better to add the field whenever we actually need it. If we added it now, it'd be just another field to set that we always set to the same thing so it'd be kind of annoying and another place for error.